frontend/.env.dev

```diff
@@ -1 +1,2 @@
 NODE_ENV=development
+VITE_LICENSE=eyJhbGciOiJSUzI1NiIsImtpZCI6InYxIiwic2F0IjoiNTYxNTQzMjUiLCJzaWQiOiIxNzMzMjY5NCIsInR5cCI6IkpXVCJ9.eyJhdWQiOiI5MjUyMzA4MCIsImlhdCI6MTY0MzQ2Mzg4MiwiaW5zdGFuY2UiOjk5OSwiaXNzIjoiYnl0ZWJhc2UiLCJwbGFuIjoiVEVBTSIsInN1YiI6Ik9SR0FOSVpBVElPTiJ9.ehieOTFNLS1XP3VLzZaT09DMdoOa24lSd-hILC6HRe9bfdn159AUaT-9GTTeZpY4blWktBxRF52xFYd5Di0wDthhpPm3_MCOxAEJ9p-oKy21_WwuplNDFlPDaYMpG_kLsK_IA51R66SD5Iid9zQgisPdMlnIO_2j22d1RQY-DKLKodzbsZLSzXtzhYMZI5ohHWWsgMbqte8Q46yQJBKh1pJQw1YGS4TaqGmwbcgHbwa_itWrT4gHOflQflTic7i33hszGAM4WaGRcKYH7OeHhsFAsjWUj4NeTo1AePorWS_iSNvsRL1oDkvc4KY6mMTZ56WgJcwhROt_7-pN0x9p7w
```

Comment:
Add a license here for dev env, with TEAM plan, 9999 instance count and unlimited expires time